lib/elixir/lib/calendar.ex

```diff
@@ -171,6 +171,7 @@ defmodule Date do
 
   """
   @spec leap_year?(Calendar.date) :: boolean()
+  def leap_year?(date)
```

Comment:
Thanks. I was trying to derive the style based on other code and it was a little ambiguous. I went with "no line when adjacent to a single function" and "line when there are multiple function heads". 

But happy to revise!